Maven release plugin doesn't work with sort plugin

**Describe the bug**

The release plugin updates the version in the pom file during the jenkins release pipeline. It will change some formating of the pom and will cause the sort plugin to fail. See related PR. https://github.com/camunda/zeebe/actions/runs/3180846815/jobs/5184904296


```
Error:  Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.27.1:check (spotless-check) on project zeebe-bom: The following files had format violations:
Error:      pom.xml
Error:          @@ -8,7 +8,7 @@
Error:           ····<artifactId>camunda-release-parent</artifactId>
Error:           ····<version>3.9.1</version>
Error:           ····<!--·do·not·remove·empty·tag·-·[http://jira.codehaus.org/browse/MNG-4687·--](http://jira.codehaus.org/browse/MNG-4687%C2%B7--)>
Error:          -····<relativePath·/>
Error:          +····<relativePath></relativePath>
Error:           ··</parent>
Error:           
Error:           ··<groupId>io.camunda</groupId>
Error:          @@ -128,7 +128,7 @@
Error:           ··········<configuration>
Error:           ············<pom>
Error:           ··············<!--·[https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#sortpom·--](https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#sortpom%C2%B7--)>
Error:          -··············<sortPom·/>
Error:          +··············<sortPom></sortPom>
Error:           ············</pom>
Error:           ··········</configuration>
Error:           ········</plugin>
Error:  Run 'mvn spotless:apply' to fix these violations.
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error: Process completed with exit code 1.
```

See related change which causes this https://github.com/camunda/zeebe/pull/10599/commits/e8e98560ab7123d3cdbdd4fdf63b5934cd251813#diff-954221b717ff3d93b2f171be5b7d728359dfa4ebee98e8e5691c0697682d247eL131



<!-- A clear and concise description of what the bug is. -->

**To Reproduce**

Build a release
<!--
Steps to reproduce the behavior

If possible add a minimal reproducer code sample
- when using the Java client: https://github.com/zeebe-io/zeebe-test-template-java

-->

**Expected behavior**
No issues with sort plugin

<!-- A clear and concise description of what you expected to happen. -->

**Log/Stacktrace**
https://github.com/camunda/zeebe/actions/runs/3180846815/jobs/5184904296
<!-- If possible add the full stacktrace or Zeebe log which contains the issue. -->

--- a/.ci/scripts/release/maven-release.sh
+++ b/.ci/scripts/release/maven-release.sh
@@ -11,5 +11,6 @@ mvn -s ${MAVEN_SETTINGS_XML} release:prepare release:perform -B \
     -DpushChanges=${PUSH_CHANGES} \
     -DremoteTagging=${PUSH_CHANGES} \
     -DlocalCheckout=${SKIP_DEPLOY} \
+    -DcompletionGoals="spotless:apply" \
     -P!autoFormat \
-    -Darguments='--settings=${MAVEN_SETTINGS_XML} -P-autoFormat -DskipChecks=true -DskipTests=true -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'
+    -Darguments='--settings=${MAVEN_SETTINGS_XML} -P-autoFormat -DskipChecks=true -DskipTests=true -Dspotless.apply.skip=false -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'

